// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/conformance/es6/computedProperties/computedPropertyNamesOnOverloads_ES6.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -5,21 +5,19 @@
 declare var accessorName: string;
 declare class C {
     [methodName](v: string): invalid;
     [methodName](): invalid;
-    [methodName](v?: string): invalid;
 }
 
 /// [Errors] ////
 
 computedPropertyNamesOnOverloads_ES6.ts(4,5): error TS1168: A computed property name in a method overload must refer to an expression whose type is a literal type or a 'unique symbol' type.
 computedPropertyNamesOnOverloads_ES6.ts(4,5): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 computedPropertyNamesOnOverloads_ES6.ts(5,5): error TS1168: A computed property name in a method overload must refer to an expression whose type is a literal type or a 'unique symbol' type.
 computedPropertyNamesOnOverloads_ES6.ts(5,5): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
-computedPropertyNamesOnOverloads_ES6.ts(6,5): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 
 
-==== computedPropertyNamesOnOverloads_ES6.ts (5 errors) ====
+==== computedPropertyNamesOnOverloads_ES6.ts (4 errors) ====
     var methodName = "method";
     var accessorName = "accessor";
     class C {
         [methodName](v: string);
@@ -32,7 +30,5 @@
 !!! error TS1168: A computed property name in a method overload must refer to an expression whose type is a literal type or a 'unique symbol' type.
         ~~~~~~~~~~~~
 !!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
         [methodName](v?: string) { }
-        ~~~~~~~~~~~~
-!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
     }
\ No newline at end of file
